site/source/components/layout/Header.tsx

```diff
@@ -63,16 +63,12 @@ export default function Header() {
 							<Switch
 								isSelected={darkMode}
 								onChange={setDarkMode}
-								role="checkbox"
-								aria-label={
-									darkMode
-										? t(
-												'navbar.deactivate-darkmode',
-												'Désactiver le mode sombre'
-										  )
-										: t('navbar.activate-darkmode', 'Activer le mode sombre')
-								}
-							/>
+								srOnlyLabel
+								/* Need this useless aria-label to silence a React-Aria warning */
+								aria-label=""
+							>
+								<Trans>Activer le mode sombre</Trans>
```

Comment:
Pour la traduction :
1. Quand c'est possible, on utilise `t()` plutôt que `<Trans>`. C'est ce que recommande [react i18n](https://react.i18next.com/latest/trans-component) :

  > While `<Trans>` gives you a lot of power by letting you interpolate or translate complex React elements, the truth is: in most cases you don't even need it.

  > **As long you have no React/HTML nodes integrated into a cohesive sentence** (text formatting like `strong`, `em`, link components, maybe others), **you won't need it** - most of the times you will be using the good old `t` function.

2. À part pour des textes très génériques ("fermer" par exemple), on précise une `i18nKey`.
  - Avec `<Trans>` cela donne `<Trans i18nKey="XXX">`
  - Avec `t` cela donne `t('XXX', 'texte à traduire')`

Ici, cela donnerait :

```suggestion
								{ t('navbar.activate-darkmode', 'Activer le mode sombre') }
```
avec plus haut :
``` ts
const { t } = useTranslation()
```